config/custom_navajo.json

```diff
@@ -18,5 +18,15 @@
     "url": "https://github.com/eutro/ocaml/archive/refs/heads/tsdnr-barriers.zip",
     "name": "5.3.0+trunk+eutro+barriers",
     "expiry": "2024-03-19"
-  }
+  },
+  {
+    "url": "https://github.com/ocaml/ocaml/archive/5.1.1.tar.gz",
+    "name": "5.1.1",
+    "expiry": "2100-01-01"
+  },
+  {
+    "url": "https://github.com/ocaml/ocaml/archive/5.1.1.tar.gz",
+    "name": "5.1.1+fp",
+    "expiry": "2100-01-01"
```

Comment:
You probably want to add something like this, here: 

    "configure": "--enable-frame-pointers",